File: scripts/verify_polish.js
import fs from 'fs';
import path from 'path';

const distHtmlPath = 'C:\\Users\\Admin\\.gemini\\antigravity-ide\\scratch\\garden-experience\\dist\\index.html';
const content = fs.readFileSync(distHtmlPath, 'utf-8');

const htmlChecks = {
  btnQuickThemeToggle: content.includes('btnQuickThemeToggle'),
  cfgAppearance: content.includes('cfg-appearance'),
  themeRadioLight: content.includes('themeRadioLight'),
  themeRadioDark: content.includes('themeRadioDark'),
  themeRadioSystem: content.includes('themeRadioSystem'),
  brandHomeLink: content.includes('brandHomeLink')
};

console.log('DIST HTML VERIFICATION RESULTS:');
Object.entries(htmlChecks).forEach(([k, v]) => {
  console.log(`  ${k}: ${v ? 'PASS' : 'FAIL'}`);
});

const srcJsPath = 'C:\\Users\\Admin\\.gemini\\antigravity-ide\\scratch\\garden-experience\\src\\main.js';
const jsContent = fs.readFileSync(srcJsPath, 'utf-8');

const jsChecks = {
  initTheme: jsContent.includes('function initTheme'),
  setThemeMode: jsContent.includes('function setThemeMode'),
  applyResolvedTheme: jsContent.includes('function applyResolvedTheme'),
  btnQuickAssign: jsContent.includes('btnQuickAssign'),
  btnQuickResolve: jsContent.includes('btnQuickResolve'),
  btnQuickWhatsapp: jsContent.includes('btnQuickWhatsapp'),
  btnQuickEmail: jsContent.includes('btnQuickEmail'),
  btnQuickInternalNote: jsContent.includes('btnQuickInternalNote'),
  localStoragePersistence: jsContent.includes("localStorage.setItem('garden_theme_mode'")
};

console.log('\nSRC JS VERIFICATION RESULTS:');
Object.entries(jsChecks).forEach(([k, v]) => {
  console.log(`  ${k}: ${v ? 'PASS' : 'FAIL'}`);
});

const cssPath = 'C:\\Users\\Admin\\.gemini\\antigravity-ide\\scratch\\garden-experience\\style.css';
const cssContent = fs.readFileSync(cssPath, 'utf-8');

const cssChecks = {
  lightThemeTokens: cssContent.includes('[data-theme="light"]'),
  kioskIsolation: cssContent.includes('#tab-kiosk'),
  quickActionsBar: cssContent.includes('.detail-quick-actions-bar'),
  smoothTransitions: cssContent.includes('transition: background-color 0.2s ease')
};

console.log('\nCSS SYSTEM VERIFICATION RESULTS:');
Object.entries(cssChecks).forEach(([k, v]) => {
  console.log(`  ${k}: ${v ? 'PASS' : 'FAIL'}`);
});

File: verify_polish.py
import os
import re

dist_html_path = r"C:\Users\Admin\.gemini\antigravity-ide\scratch\garden-experience\dist\index.html"
with open(dist_html_path, "r", encoding="utf-8") as f:
    content = f.read()

checks = {
    "btnQuickThemeToggle": "btnQuickThemeToggle" in content,
    "cfg-appearance": "cfg-appearance" in content,
    "themeRadioLight": "themeRadioLight" in content,
    "themeRadioDark": "themeRadioDark" in content,
    "themeRadioSystem": "themeRadioSystem" in content,
    "brandHomeLink": "brandHomeLink" in content
}

print("DIST HTML VERIFICATION RESULTS:")
for k, v in checks.items():
    print(f"  {k}: {'PASS' if v else 'FAIL'}")

src_js_path = r"C:\Users\Admin\.gemini\antigravity-ide\scratch\garden-experience\src\main.js"
with open(src_js_path, "r", encoding="utf-8") as f:
    js_content = f.read()

js_checks = {
    "initTheme": "function initTheme" in js_content,
    "setThemeMode": "function setThemeMode" in js_content,
    "applyResolvedTheme": "function applyResolvedTheme" in js_content,
    "btnQuickAssign": "btnQuickAssign" in js_content,
    "btnQuickResolve": "btnQuickResolve" in js_content,
    "btnQuickWhatsapp": "btnQuickWhatsapp" in js_content,
    "btnQuickEmail": "btnQuickEmail" in js_content,
    "btnQuickInternalNote": "btnQuickInternalNote" in js_content,
    "localStorage persistence": "localStorage.setItem('garden_theme_mode'" in js_content
}

print("\nSRC JS VERIFICATION RESULTS:")
for k, v in js_checks.items():
    print(f"  {k}: {'PASS' if v else 'FAIL'}")
